inventory_management/inventory_management/report/stock_balance/stock_balance.py

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) 2024, Viny Selopal and contributors
+# For license information, please see license.txt
+
+import frappe
+from frappe import _
+
+
+def execute(filters=None):
+    columns = get_columns()
+    data = get_data(filters)
+
+    return columns, data
+
+
+def get_columns():
+    columns = [
```

Comment:
<img width="1292" alt="image" src="https://github.com/vinyselopal/InventoryManagementFrappe/assets/24353136/2d0dbc7f-9e5a-4305-803e-8ff9f8421b35">

The Stock Balance report should show me a consolidated balance for an Item-Warehouse combination. The current implementation is just a copy of stock ledger report.

Check this report from ERPNext. You can ignore the In Qty Out Qty columns for simplicity, but just this basic consolidation is required in the report based on date ranges. So that I can go back and check what was the item balance in a particular warehouse on a given date

<img width="1292" alt="image" src="https://github.com/vinyselopal/InventoryManagementFrappe/assets/24353136/5c39514d-9ee8-4883-a1e7-d7b177507fad">